CRM/Core/Payment/Form.php

```diff
@@ -326,6 +326,27 @@ public static function setDefaultValues(&$form, $contactID) {
       $form->_defaults["billing_state_province_id-{$form->_bltID}"] = CRM_Core_Config::singleton()
         ->defaultContactStateProvince;
     }
+
+    // In backoffice edit form, retrieve pan_truncation and card type of corresponding entity from latest payment,
+    //  if found that set in default values. However if the payment was done using any payment processor either via
+    //  backoffice credit card or online registration, freeze these two fields so user cannot edit their corresponding values
+    if (!empty($form->isBackOffice) && !empty($form->_entityID)) {
```

Comment:
At least this won't cause any regression and is required by IIDA client. So what if we allow it in 4.7.19 and handle it in next phase may be 4.7.20. We could leave a todo note here as reminder. What ya say ?